revolut/__init__.py

```diff
@@ -141,22 +141,18 @@ def __init__(self, token, device_id):
                     'Authorization': 'Basic '+token,
                     }
 
-    def _get(self, url, expected_status_code=200):
-        ret = self.session.get(url=url, headers=self.headers)
+    def _get(self, url, *, expected_status_code=200, **kwargs):
+        ret = self.session.get(url=url, **kwargs)
         if ret.status_code != expected_status_code:
             raise ConnectionError(
-                'Status code {status} for url {url}\n{content}'.format(
-                    status=ret.status_code, url=url, content=ret.text))
+                f'Status code {ret.status_code} for url {url}\n{ret.text}')
```

Comment:
Could you please keep the `format`-styled strings instead of f-strings?
Otherwise users will require Python >= 3.7.